decoder/vaapidecoder_vc1.cpp

```diff
@@ -355,20 +383,32 @@ YamiStatus VaapiDecoderVC1::decode(uint8_t* data, uint32_t size, uint64_t pts)
         return YAMI_FAIL;
     }
 
-    if ((m_parser.m_frameHdr.picture_type == FRAME_I)
-        || (m_parser.m_frameHdr.picture_type == FRAME_P)
-        || (m_parser.m_frameHdr.picture_type == FRAME_SKIPPED)) {
-        m_forwardPicture = picture;
+    if (m_parser.m_frameHdr.picture_type == FRAME_B)
+        picture->m_isReference = false;
+    if (m_dpbIdx == 2) {
+        if (!picture->m_isReference) {
+            outputPicture(m_dpb[0]);
```

Comment:
could we name 0, as FORWARD_PICTURE,  and 1 as BACKWORD_PICTURE? It will make code more readable